edx_django_utils/plugins/docs/decisions/0003-plugin-contexts.rst

```diff
@@ -0,0 +1,88 @@
+Plugin Contexts
```

Comment:
I don't know why this was named `0003-...` in edx-platform.  Let's rename to `0001`.